### ISSUE TITLE
Chinese char can't be copyed and searched in outport PDF

### ISSUE THREAD

**Issue by xdxdy** [MacDownApp/macdown#627]
![f3ae9cea-7096-41f5-bea6-8a61f5e755f3](https://cloud.githubusercontent.com/assets/11024400/16553724/d55cae9e-41fc-11e6-9ef0-51c1cb9f516a.png)

just like this， i use this to outport

<img width="298" alt="2016-07-04 3 35 19" src="https://cloud.githubusercontent.com/assets/11024400/16553756/03efbc1a-41fd-11e6-9224-c86a590356c9.png">

**Comment by franklinyu** [comment]
This seems to be some encoding issue.

**Comment by uranusjr** [comment]
Could you provide an example document for me to export? I just tried creating a new document with one line of Chinese, and the PDF export seems to copy fine.

**Comment by franklinyu** [comment]
test.md:

```markdown
测试
```

When I copy from the PDF file, it gives

```text
􁁥􁦶
```

**Comment by uranusjr** [comment]
I tried creating a new document and export (⌘-opt-p). The PDF file works fine. Could you provide your PDF export so I can check whether it works?

**Comment by franklinyu** [comment]
Yes sure: [test.pdf](https://github.com/MacDownApp/macdown/files/428015/test.pdf)

**Comment by uranusjr** [comment]
Hmm yes I can reproduce your problem now. Guess I’ll need to crack open the PDF.
